packages/wrangler/src/pages/dev.tsx

```diff
@@ -385,6 +385,9 @@ export const Handler = async ({
 					define: {
 						__ROUTES__: routes,
 					},
+					inject: [
```

Comment:
I blame you @GregBrimble, because you told me abt it, and now I can't un-know it ;)